x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -31,6 +31,9 @@
             "__index": {
               "type": "long"
             },
+            "__swimlane": {
+              "type": "long"
+            },
             "__pagerduty": {
```

Comment:
if you want to add the telemetry for a new action type, there are a few more places that need to be updated:
`x-pack/plugins/actions/server/usage/actions_usage_collector.ts` and `elastic/telemetry` repository like [here](https://github.com/elastic/telemetry/pull/530/files)

Reply:
Thanks for pointing this out @YulNaumenko ! I created a PR here for the telemetry repo: https://github.com/elastic/telemetry/pull/537